tests/test_server.py

```diff
@@ -74,23 +83,31 @@ def test_server_parse_channel_data(server, rtm_start_fixture):
 def test_server_parse_user_data(server, rtm_start_fixture):
     server.parse_user_data(rtm_start_fixture["users"])
     # Find user by Name
-    userbyname = server.users.find('fakeuser')
-    assert type(userbyname) == User
-    assert userbyname == "fakeuser"
-    assert userbyname != "someotheruser"
+    user_by_name = server.users.find('fakeuser')
+    assert type(user_by_name) == User
+    assert user_by_name == "fakeuser"
```

Comment:
Users are coerced into a string? is that a useful feature?

Reply:
I'm especially worried about this because we'd like to de-emphasize programmatic usage of username.

Reply:
this is from a refactor of the "SearchList" class that allowed searching through the client's cache by username or user ID. I just cleaned up the names for readability.